fairseq/progress_bar.py

```diff
@@ -225,15 +225,15 @@ def __init__(self, wrapped_bar, tensorboard_logdir, args):
             self._writers = {}
         except ImportError:
             print("tensorboard or required dependencies not found, "
-                  "please see README for using tensorboard.")
+                  "please see README for using tensorboard. (e.g. pip install tensorboardX)")
             self.SummaryWriter = None
 
     def _writer(self, key):
         if self.SummaryWriter is None:
             return None
         if key not in self._writers:
             self._writers[key] = self.SummaryWriter(
-                log_dir=os.path.join(self.tensorboard_logdir, key),
+                logdir=os.path.join(self.tensorboard_logdir, key),
```

Comment:
Won't this break things for anyone using an old version of tensorboardX? It's not elegant, but let's remove the kwarg so that it works either way.

Reply:
Thank you! You are absolutely right. I just removed the kwarg in the latest commit.